api_spec.yaml

```diff
@@ -1266,17 +1472,285 @@ paths:
+                                    - type: object
+                                      properties:
+                                          status:
+                                              type: string
+                                              default: "ASSOCIATION_NOT_ALLOWED_ERROR"
+                                          reason:
+                                              type: string
+
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+    /dashboard/api/tenants/user/disassociate:
+        put:
+            tags:
+                - Multitenancy
+            summary: Disassociate a user from a tenant
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                tenantId:
+                                    type: string
+                                userId:
+                                    type: string
+            responses:
+                200:
+                    description: Success
+                    content:
+                        application/json:
+                            schema:
+                                type: object
+                                properties:
+                                    status:
+                                        type: string
+                                        default: "OK"
+                                    wasAssociated:
+                                        type: boolean
+                                        default: true
+
+                401:
+                    description: Unauthorised access
+                    content:
+                        text/plain:
+                            schema:
+                                type: string
+                                enum:
+                                    - Unauthorised access
+    /dashboard/api/tenants/third-party:
+        put:
+            tags:
+                - Multitenancy
+            summary: Create or update third party config for a tenant
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            requestBody:
```

Comment:
tenantId should not be part of the body, it should be read from url path

Reply:
Done